server/models/snippet.js

```diff
@@ -63,7 +63,7 @@ module.exports = function(sequelize, DataTypes) {
       }
     },
     instanceMethods: {
-      toJson: function () {
+      toJson: function (currentUserId) {
```

Comment:
to return only current user's rating instead of all of them. if currentUserId is null (not given in call) it still works, returning '0' as current user's rating @blimusiek